### PR TITLE
GPGME health integration

### PR DESCRIPTION
**Pull request by awisema**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

Support GPG decryption of Health imports

## Type of change
[//]: # 'remove options that are not relevant'
- [X] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

**Review by eanders — APPROVED** [review]
@awisema looks reasonable to me!  Thanks!

File: app/models/health/tasks/import_epic.rb [hunk 1]
```diff
@@ -164,7 +176,14 @@ def fetch_files
       notify 'Health data downloaded'
     end
 
-    def read_csv_file path:
+    def read_csv_file(path:)
+      # Decrypt the file if it is '*.gpg'
+      if path =~ /.gpg$/
```
Comment (eanders) [inline app/models/health/tasks/import_epic.rb]: maybe?
```suggestion
      if path =~ File.extname(path) == '.gpg'
```

File: app/models/health/tasks/import_epic.rb [hunk 2]
```diff
@@ -164,7 +176,14 @@ def fetch_files
...
+        data = @config.decrypt(File.read(path))
+        path = path[...-4] # Trim off '.gpg'
```
Comment (eanders) [inline app/models/health/tasks/import_epic.rb]: I feel like there must be a more ruby way to do this, but I'm not finding a great one.  something like `path.sub(/.gpg$//)`  (I don't think that's correct though.)

Comment (awisema) [reply app/models/health/tasks/import_epic.rb]: How about `File.join(File.dirname(path), File.basename(path, '.gpg'))`?

Comment (eanders) [reply app/models/health/tasks/import_epic.rb]: Yep, that's the most Rails-y I think!

**Pull request merged**